Dfe.Academies.External.Web/Pages/School/LoanDetails.cshtml.cs

```diff
@@ -0,0 +1,121 @@
+﻿using System.ComponentModel.DataAnnotations;
+using Dfe.Academies.External.Web.Pages.Base;
+using Dfe.Academies.External.Web.Services;
+using Dfe.Academies.External.Web.ViewModels;
+using Microsoft.AspNetCore.Mvc;
+using Microsoft.AspNetCore.Mvc.RazorPages;
+
+namespace Dfe.Academies.External.Web.Pages.School
+{
+	public class LoanDetails : BasePageEditModel
+	{
+		[BindProperty]
+		public int  Id { get; set; }
+		[BindProperty]
+		public int ApplicationId { get; set; }
+
+		[BindProperty]
+		public int Urn { get; set; }
+		[BindProperty]
```

Comment:
the validation text isn't as per the ticket if you don't fill these in. just spotting before dan does